app/controllers/bibliographic_controller.rb

```diff
@@ -158,7 +170,10 @@ def voyager_opts
     # @param opts [Hash] optional arguments
     # @return [Array<Object>] the set of bib. records
     def find_by_id(id, opts)
-      VoyagerHelpers::Liberator.get_bib_record(id, nil, opts)
+      bib_adapter.get_bib_record(sanitize(params[:bib_id]), nil, opts)
```

Comment:
I will update it. Now I looked more & there are 2 methods doing the same thing. This will be part of a refactor. https://github.com/pulibrary/marc_liberation/blob/493949186cbb8cf5f6bb888aced888cfca149051/app/controllers/bibliographic_controller.rb#L203 and https://github.com/pulibrary/marc_liberation/blob/493949186cbb8cf5f6bb888aced888cfca149051/app/controllers/bibliographic_controller.rb#L142